res/skins/LateNight/fx_samplers_container_full.xml

```diff
@@ -6,7 +6,7 @@
         <Children>
             <WidgetStack currentpage="[Library],show_big_current" persist="true">
                 <Children>
-                    <WidgetGroup trigger="[Library],unshow_big">
+                    <WidgetGroup trigger="[Master],maximize_library">
```

Comment:
I`ll take a look.
Seems to work ok for now, but will test a bit more.

There is a logic error with the skin (in master), where you can end up with no widgets at all. Toggle the ``Library`` button and then ``Big Library``. Not yet sure how to solve that.